<!doctype html><html lang=en-us><head><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><meta http-equiv=x-ua-compatible content="IE=edge"><meta name=generator content="Source Themes Academic 4.5.0"><meta name=author content="Swapnil Mishra"><meta name=description content="Assistant Professor"><link rel=alternate hreflang=en-us href=https://s-mishra.github.io/><meta name=theme-color content="#2962ff"><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/academicons/1.8.6/css/academicons.min.css integrity="sha256-uFVgMKfistnJAfoCUQigIl+JfUaP47GrRKjf6CTPVmw=" crossorigin=anonymous><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.11.2/css/all.min.css integrity="sha256-+N4/V/SbAFiW1MPBCXnfnP9QSN3+Keu+NlB+0ev/YKQ=" crossorigin=anonymous><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.css integrity="sha256-Vzbj7sDDS/woiFS3uNKo8eIuni59rjyNGtXfstRzStA=" crossorigin=anonymous><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.10/styles/github.min.css crossorigin=anonymous title=hl-light><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.10/styles/dracula.min.css crossorigin=anonymous title=hl-dark disabled><link rel=stylesheet href="https://fonts.googleapis.com/css?family=Montserrat:400,700%7CRoboto:400,400italic,700%7CRoboto+Mono&display=swap"><link rel=stylesheet href=/css/academic.css><script async src="https://www.googletagmanager.com/gtag/js?id=UA-118959441-1"></script>
<script>window.dataLayer=window.dataLayer||[];function gtag(){dataLayer.push(arguments)}function trackOutboundLink(e){gtag("event","click",{event_category:"outbound",event_label:e,transport_type:"beacon",event_callback:function(){document.location=e}}),console.debug("Outbound link clicked: "+e)}function onClickCallback(e){if(e.target.tagName!=="A"||e.target.host===window.location.host)return;trackOutboundLink(e.target)}gtag("js",new Date),gtag("config","UA-118959441-1",{}),document.addEventListener("click",onClickCallback,!1)</script><link rel=alternate href=/index.xml type=application/rss+xml title="Swapnil Mishra"><link rel=manifest href=/index.webmanifest><link rel=icon type=image/png href=/img/icon-32.png><link rel=apple-touch-icon type=image/png href=/img/icon-192.png><link rel=canonical href=https://s-mishra.github.io/><meta property="twitter:card" content="summary_large_image"><meta property="twitter:site" content="@creswapi"><meta property="twitter:creator" content="@creswapi"><meta property="og:site_name" content="Swapnil Mishra"><meta property="og:url" content="https://s-mishra.github.io/"><meta property="og:title" content="Swapnil Mishra"><meta property="og:description" content="Assistant Professor"><meta property="og:image" content="https://s-mishra.github.io/img/portrait.jpg"><meta property="twitter:image" content="https://s-mishra.github.io/img/portrait.jpg"><meta property="og:locale" content="en-us"><meta property="og:updated_time" content="2022-01-01T00:00:00+00:00"><script type=application/ld+json>{"@context":"https://schema.org","@type":"WebSite","potentialAction":{"@type":"SearchAction","target":"https://s-mishra.github.io/?q={search_term_string}","query-input":"required name=search_term_string"},"url":"https://s-mishra.github.io/"}</script><title>Swapnil Mishra</title></head><body id=top data-spy=scroll data-offset=70 data-target=#navbar-main><aside class=search-results id=search><div class=container><section class=search-header><div class="row no-gutters justify-content-between mb-3"><div class=col-6><h1>Search</h1></div><div class="col-6 col-search-close"><a class=js-search href=#><i class="fas fa-times-circle text-muted" aria-hidden=true></i></a></div></div><div id=search-box><input name=q id=search-query placeholder=Search... autocapitalize=off autocomplete=off autocorrect=off spellcheck=false type=search></div></section><section class=section-search-results><div id=search-hits></div></section></div></aside><nav class="navbar navbar-light fixed-top navbar-expand-lg py-0 compensate-for-scrollbar" id=navbar-main><div class=container><a class=navbar-brand href=/>Swapnil Mishra</a>
<button type=button class=navbar-toggler data-toggle=collapse data-target=#navbar aria-controls=navbar aria-expanded=false aria-label="Toggle navigation">
<span><i class="fas fa-bars"></i></span></button><div class="collapse navbar-collapse" id=navbar><ul class="navbar-nav mr-auto"><li class=nav-item><a class=nav-link href=/#about data-target=#about><span>Home</span></a></li><li class=nav-item><a class=nav-link href=/#featured data-target=#featured><span>Featured</span></a></li><li class=nav-item><a class=nav-link href=/#publications data-target=#publications><span>Publications</span></a></li><li class=nav-item><a class=nav-link href=/#grants data-target=#grants><span>Grants</span></a></li><li class=nav-item><a class=nav-link href=/#teaching data-target=#teaching><span>Teaching</span></a></li><li class=nav-item><a class=nav-link href=/files/cv.pdf><span>CV</span></a></li><li class=nav-item><a class=nav-link href=/#contact data-target=#contact><span>Contact</span></a></li></ul><ul class="navbar-nav ml-auto"><li class=nav-item><a class="nav-link js-search" href=#><i class="fas fa-search" aria-hidden=true></i></a></li><li class=nav-item><a class="nav-link js-dark-toggle" href=#><i class="fas fa-moon" aria-hidden=true></i></a></li></ul></div></div></nav><span class="js-widget-page d-none"></span><section id=about class="home-section wg-about"><div class=container><div class=row><div class="col-12 col-lg-4"><div id=profile><img class=portrait src=/authors/admin/avatar_hu238cf5d28403586e38665a841702230f_792414_250x250_fill_q90_lanczos_center.jpg alt=Avatar><div class=portrait-title><h2>Swapnil Mishra</h2><h3>Assistant Professor</h3><h3><a href=https://sph.nus.edu.sg/ target=_blank rel=noopener><span>National University of Singapore</span></a></h3></div><ul class=network-icon aria-hidden=true><li><a href=mailto:swapnilmishra%20dot%20anu%20at%20gmail%20dot%20com><i class="fas fa-envelope big-icon"></i></a></li><li><a href=https://twitter.com/creswapi target=_blank rel=noopener><i class="fab fa-twitter big-icon"></i></a></li><li><a href="https://scholar.google.co.uk/citations?user=RqbpaXcAAAAJ&hl=en" target=_blank rel=noopener><i class="ai ai-google-scholar big-icon"></i></a></li><li><a href=https://orcid.org/0000-0002-8759-5902 target=_blank rel=noopener><i class="ai ai-orcid big-icon"></i></a></li><li><a href=https://github.com/s-mishra target=_blank rel=noopener><i class="fab fa-github big-icon"></i></a></li><li><a href=/files/cv.pdf><i class="ai ai-cv big-icon"></i></a></li></ul></div></div><div class="col-12 col-lg-8"><h1>Biography</h1><p>I am an Assistant Professor at <a href=https://sph.nus.edu.sg/>National University of Singapore (NUS)</a>, where I am primarily working at intersection of public health, machine learning and Bayesian modelling. I am part of <a href=https://mlgh.net>Machine Learning & Global Health Network</a>, which is a multi-country and multi-organisation network focused on doing fundamental research in machine learning and problems related to global health.</p><p>My research focuses on applying and developing statistical machine learning techniques for the broader and messier world of science and public policy, especially global health. I develop flexible and scalable models for understanding various spatiotemporal data, for example, epidemics (COVID-19, Malaria, HIV) and crime. For my doctorate, I built models for understanding the evolution of popularity in social media. My work focused on algorithms to model point processes with classical machine learning techniques as well as using modern deep learning networks, mainly recurrent networks.</p><div class=row><div class=col-md-5><h3>Interests</h3><ul class=ul-interests><li>Infectious Disease Modelling</li><li>Hierarchial Bayesian Modelling</li><li>Renewal Processes</li><li>Online Information Diffusion</li><li>Stochastic Point Processes</li><li>Generative Deep Learning</li></ul></div><div class=col-md-7><h3>Education</h3><ul class="ul-edu fa-ul"><li><i class="fa-li fas fa-graduation-cap"></i><div class=description><p class=course>PhD in Machine Learning, 2019</p><p class=institution>The Australian National University</p></div></li><li><i class="fa-li fas fa-graduation-cap"></i><div class=description><p class=course>MS(Hons.) in Artificial Intelligence, 2014</p><p class=institution>The Australian National University</p></div></li><li><i class="fa-li fas fa-graduation-cap"></i><div class=description><p class=course>BE in Computer Engineering, 2009</p><p class=institution>Maharashtra Institute of Technology</p></div></li></ul></div></div></div></div></div></section><section id=featured class="home-section wg-featured"><div class=container><div class=row><div class="col-12 col-lg-4 section-heading"><h1>Featured Publications</h1></div><div class="col-12 col-lg-8"><div class=card-simple><div class=article-metadata><div><span><a href=/authors/swapnil-mishra/>Swapnil Mishra</a></span>, <span><a href=/authors/seth-flaxman/>Seth Flaxman</a></span>, <span><a href=/authors/tresnia-berah/>Tresnia Berah</a></span>, <span><a href=/authors/harrison-zhu/>Harrison Zhu</a></span>, <span><a href=/authors/mikko-pakkanen/>Mikko Pakkanen</a></span>, <span><a href=/authors/samir-bhatt/>Samir Bhatt</a></span></div><span class=article-date>January 2022</span>
<span class=middot-divider></span>
<span class=pub-publication><em>Statistics and Computing</em></span></div><a href=/publication/mishra-2022-pi/><img src=/publication/mishra-2022-pi/featured_hu02e321a026220c0d8b5d6e412a8976f1_46508_918x517_fill_q90_lanczos_smart1_3.png class=article-banner alt></a><h3 class="article-title mb-1 mt-3"><a href=/publication/mishra-2022-pi/>π VAE: a stochastic process prior for Bayesian deep learning with MCMC</a></h3><div class=article-style><p>Stochastic processes provide a mathematically elegant way to model complex data. In theory, they provide flexible priors over function classes that can encode a wide range of interesting assumptions. However, in practice efficient inference by optimisation or marginalisation is difficult, a problem further exacerbated with big data and high dimensional input spaces. We propose a novel variational autoencoder (VAE) called the prior encoding variational autoencoder (𝜋VAE). 𝜋VAE is a new continuous stochastic process. We use 𝜋VAE to learn low dimensional embeddings of function classes by combining a trainable feature mapping with generative model using a VAE. We show that our framework can accurately learn expressive function classes such as Gaussian processes, but also properties of functions such as their integrals. For popular tasks, such as spatial interpolation, 𝜋VAE achieves state-of-the-art performance both in terms of accuracy and computational efficiency. Perhaps most usefully, we demonstrate an elegant and scalable means of performing fully Bayesian inference for stochastic processes within probabilistic programming languages such as Stan.</p></div><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://link.springer.com/content/pdf/10.1007/s11222-022-10151-w.pdf target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/mishra-2022-pi/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://github.com/MLGlobalHealth/pi-vae%20 target=_blank rel=noopener>Code</a>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://github.com/MLGlobalHealth/pi-vae%20 target=_blank rel=noopener>Dataset</a>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1007/s11222-022-10151-w target=_blank rel=noopener>DOI</a>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://link.springer.com/article/10.1007/s11222-022-10151-w#appendices target=_blank rel=noopener>Supplementary Material</a></div></div><div class=card-simple><div class=article-metadata><div><span><a href=/authors/swapnil-mishra/>Swapnil Mishra*</a></span>, <span><a href=/authors/nuno-r.-faria/>Nuno R. Faria*</a></span>, <span><a href=/authors/thomas-a.-mellan/>Thomas A. Mellan*</a></span>, <span><a href=/authors/charles-whittaker/>Charles Whittaker*</a></span>, <span><a href=/authors/ingra-m.-claro/>Ingra M. Claro*</a></span>, <span><a href=/authors/darlan-da-s.-candido/>Darlan da S. Candido*</a></span>, <span><a href=/authors/others/>others</a></span>, <span><a href=/authors/oliver-g.-pybus/>Oliver G. Pybus‡</a></span>, <span><a href=/authors/seth-flaxman/>Seth Flaxman‡</a></span>, <span><a href=/authors/samir-bhatt/>Samir Bhatt‡</a></span>, <span><a href=/authors/ester-c.-sabino/>Ester C. Sabino‡</a></span></div><span class=article-date>April 2021</span>
<span class=middot-divider></span>
<span class=pub-publication>Science (2021)</span></div><a href=/publication/science-p1/><img src=/publication/science-p1/featured_hu1277959bc9d8bf7b63da181e7deac72f_92663_918x517_fill_q90_lanczos_smart1.jpg class=article-banner alt></a><h3 class="article-title mb-1 mt-3"><a href=/publication/science-p1/>Genomics and epidemiology of the P.1 SARS-CoV-2 lineage in Manaus, Brazil</a></h3><div class=article-style><p>Cases of SARS-CoV-2 infection in Manaus, Brazil, resurged in late 2020, despite previously high levels of infection. Genome sequencing of viruses sampled in Manaus between November 2020 and January 2021 revealed the emergence and circulation of a novel SARS-CoV-2 variant of concern. Lineage P.1, acquired 17 mutations, including a trio in the spike protein (K417T, E484K and N501Y) associated with increased binding to the human ACE2 receptor. Molecular clock analysis shows that P.1 emergence occurred around mid-November 2020 and was preceded by a period of faster molecular evolution. Using a two-category dynamical model that integrates genomic and mortality data, we estimate that P.1 may be 1.7–2.4-fold more transmissible, and that previous (non-P.1) infection provides 54–79% of the protection against infection with P.1 that it provides against non-P.1 lineages. Enhanced global genomic surveillance of variants of concern, which may exhibit increased transmissibility and/or immune evasion, is critical to accelerate pandemic responsiveness.</p></div><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://science.sciencemag.org/content/early/2021/04/13/science.abh2644.full.pdf target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/science-p1/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://github.com/CADDE-CENTRE/Novel-SARS-CoV-2-P1-Lineage-in-Brazil target=_blank rel=noopener>Code</a>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://github.com/CADDE-CENTRE/Novel-SARS-CoV-2-P1-Lineage-in-Brazil target=_blank rel=noopener>Dataset</a>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://science.sciencemag.org/content/early/2021/04/13/science.abh2644 target=_blank rel=noopener>Source Document</a>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1126/science.abh2644 target=_blank rel=noopener>DOI</a>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://science.sciencemag.org/content/sci/suppl/2021/04/13/science.abh2644.DC1/abh2644-Faria-SM.pdf target=_blank rel=noopener>Supplementary Material</a></div></div><div class=card-simple><div class=article-metadata><div><span><a href=/authors/swapnil-mishra/>Swapnil Mishra*</a></span>, <span><a href=/authors/erik-volz/>Erik Volz*</a></span>, <span><a href=/authors/meera-chand/>Meera Chand*</a></span>, <span><a href=/authors/jeffrey-c.-barrett/>Jeffrey C. Barrett*</a></span>, <span><a href=/authors/robert-johnson/>Robert Johnson*</a></span>, <span><a href=/authors/lily-geidelberg/>Lily Geidelberg</a></span>, <span><a href=/authors/wes-r.-hinsley/>Wes R. Hinsley</a></span>, <span><a href=/authors/daniel-j.-laydon/>Daniel J. Laydon</a></span>, <span><a href=/authors/gavin-dabrera/>Gavin Dabrera</a></span>, <span><a href=/authors/aine-otoole/>Áine O’Toole</a></span>, <span><a href=/authors/roberto-amato/>Roberto Amato</a></span>, <span><a href=/authors/manon-ragonnet-cronin/>Manon Ragonnet-Cronin</a></span>, <span><a href=/authors/ian-harrison/>Ian Harrison</a></span>, <span><a href=/authors/ben-jackson/>Ben Jackson</a></span>, <span><a href=/authors/cristina-v.-ariani/>Cristina V. Ariani</a></span>, <span><a href=/authors/olivia-boyd/>Olivia Boyd</a></span>, <span><a href=/authors/nicholas-j.-loman/>Nicholas J. Loman</a></span>, <span><a href=/authors/john-t.-mccrone/>John T. McCrone</a></span>, <span><a href=/authors/sonia-goncalves/>Sónia Gonçalves</a></span>, <span><a href=/authors/david-jorgensen/>David Jorgensen</a></span>, <span><a href=/authors/richard-myers/>Richard Myers</a></span>, <span><a href=/authors/verity-hill/>Verity Hill</a></span>, <span><a href=/authors/david-k.-jackson/>David K. Jackson</a></span>, <span><a href=/authors/katy-gaythorpe/>Katy Gaythorpe</a></span>, <span><a href=/authors/natalie-groves/>Natalie Groves</a></span>, <span><a href=/authors/john-sillitoe/>John Sillitoe</a></span>, <span><a href=/authors/dominic-p.-kwiatkowski/>Dominic P. Kwiatkowski</a></span>, <span><a href=/authors/the-covid-19-genomics-uk-cog-uk-consortium/>The COVID-19 Genomics UK (COG-UK) consortium</a></span>, <span><a href=/authors/seth-flaxman/>Seth Flaxman</a></span>, <span><a href=/authors/oliver-ratmann/>Oliver Ratmann</a></span>, <span><a href=/authors/samir-bhatt/>Samir Bhatt</a></span>, <span><a href=/authors/susan-hopkins/>Susan Hopkins</a></span>, <span><a href=/authors/axel-gandy/>Axel Gandy*</a></span>, <span><a href=/authors/andrew-rambaut/>Andrew Rambaut*</a></span>, <span><a href=/authors/neil-m.-ferguson/>Neil M. Ferguson*</a></span></div><span class=article-date>March 2021</span>
<span class=middot-divider></span>
<span class=pub-publication>Nature (2021)</span></div><a href=/publication/nature21/><img src=/publication/nature21/featured_hu67ec5fc1f6eff630e08eb060340f426d_80695_918x517_fill_q90_lanczos_smart1.jpg class=article-banner alt></a><h3 class="article-title mb-1 mt-3"><a href=/publication/nature21/>Assessing transmissibility of SARS-CoV-2 lineage B.1.1.7 in England</a></h3><div class=article-style><p>The SARS-CoV-2 lineage B.1.1.7, designated a Variant of Concern 202012/01 (VOC) by Public Health England1, originated in the UK in late Summer to early Autumn 20202. Whole genome SARS-CoV-2 sequence data collected from community-based diagnostic testing shows an unprecedentedly rapid expansion of the B.1.1.7 lineage during Autumn 2020, suggesting a selective advantage. We find that changes in VOC frequency inferred from genetic data correspond closely to changes inferred by S-gene target failures (SGTF) in community-based diagnostic PCR testing. Analysis of trends in SGTF and non-SGTF case numbers in local areas across England shows that the VOC has higher transmissibility than non-VOC lineages, even if the VOC has a different latent period or generation time. The SGTF data indicate a transient shift in the age composition of reported cases, with a larger share of under 20 year olds among reported VOC than non-VOC cases. Time-varying reproduction numbers for the VOC and cocirculating lineages were estimated using SGTF and genomic data. The best supported models did not indicate a substantial difference in VOC transmissibility among different age groups. There is a consensus among all analyses that the VOC has a substantial transmission advantage with a 50% to 100% higher reproduction number.</p></div><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://www.nature.com/articles/s41586-021-03470-x.pdf target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/nature21/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://github.com/mrc-ide/sarscov2-b.1.1.7 target=_blank rel=noopener>Code</a>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://github.com/mrc-ide/sarscov2-b.1.1.7 target=_blank rel=noopener>Dataset</a>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://www.nature.com/articles/s41586-021-03470-x target=_blank rel=noopener>Source Document</a>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-021-03470-x/MediaObjects/41586_2021_3470_MOESM1_ESM.pdf target=_blank rel=noopener>Supplementary Material</a></div></div><div class=card-simple><div class=article-metadata><div><span><a href=/authors/swapnil-mishra/>Swapnil Mishra*</a></span>, <span><a href=/authors/seth-flaxman/>Seth Flaxman*</a></span>, <span><a href=/authors/axel-gandy/>Axel Gandy*</a></span>, <span><a href=/authors/h.-juliette-t.-unwin/>H. Juliette T. Unwin</a></span>, <span><a href=/authors/thomas-a.-mellan/>Thomas A. Mellan</a></span>, <span><a href=/authors/helen-coupland/>Helen Coupland</a></span>, <span><a href=/authors/charles-whittaker/>Charles Whittaker</a></span>, <span><a href=/authors/harrison-zhu/>Harrison Zhu</a></span>, <span><a href=/authors/tresnia-berah/>Tresnia Berah</a></span>, <span><a href=/authors/jeffrey-w.-eaton/>Jeffrey W. Eaton</a></span>, <span><a href=/authors/melodie-monod/>Mélodie Monod</a></span>, <span><a href=/authors/imperial-college-covid-19-response-team/>Imperial College COVID-19 Response Team</a></span>, <span><a href=/authors/azra-c.-ghani/>Azra C. Ghani</a></span>, <span><a href=/authors/christl-a.-donnelly/>Christl A. Donnelly</a></span>, <span><a href=/authors/steven-m.-riley/>Steven M. Riley</a></span>, <span><a href=/authors/michaela-a.-c.-vollmer/>Michaela A. C. Vollmer</a></span>, <span><a href=/authors/neil-m.-ferguson/>Neil M. Ferguson</a></span>, <span><a href=/authors/lucy-c.-okell/>Lucy C. Okell</a></span>, <span><a href=/authors/samir-bhatt/>Samir Bhatt*</a></span></div><span class=article-date>June 2020</span>
<span class=middot-divider></span>
<span class=pub-publication>Nature (2020)</span></div><a href=/publication/nature-20/><img src=/publication/nature-20/featured_hu86dc334a7c3ca3d5ab46820c93a40d2f_184935_918x517_fill_q90_lanczos_smart1_3.png class=article-banner alt></a><h3 class="article-title mb-1 mt-3"><a href=/publication/nature-20/>Estimating the effects of non-pharmaceutical interventions on COVID-19 in Europe.</a></h3><div class=article-style><p>Following the emergence of a novel coronavirus (SARS-CoV-2) and its spread outside of China, Europe has experienced large epidemics. In response, many European countries have implemented unprecedented non-pharmaceutical interventions such as closure of schools and national lockdowns. We study the impact of major interventions across 11 European countries for the period from the start of COVID-19 until the 4th of May 2020 when lockdowns started to be lifted. Our model calculates backwards from observed deaths to estimate transmission that occurred several weeks prior, allowing for the time lag between infection and death. We use partial pooling of information between countries with both individual and shared effects on the reproduction number. Pooling allows more information to be used, helps overcome data idiosyncrasies, and enables more timely estimates. Our model relies on fixed estimates of some epidemiological parameters such as the infection fatality rate, does not include importation or subnational variation and assumes that changes in the reproduction number are an immediate response to interventions rather than gradual changes in behavior. Amidst the ongoing pandemic, we rely on death data that is incomplete, with systematic biases in reporting, and subject to future consolidation. We estimate that, for all the countries we consider, current interventions have been sufficient to drive the reproduction number Rt below 1 (probability Rt&lt; 1.0 is 99.9%) and achieve epidemic control. We estimate that, across all 11 countries, between 12 and 15 million individuals have been infected with SARS-CoV-2 up to 4th May, representing between 3.2% and 4.0% of the population. Our results show that major non-pharmaceutical interventions and lockdown in particular have had a large effect on reducing transmission. Continued intervention should be considered to keep transmission of SARS-CoV-2 under control.</p></div><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://www.nature.com/articles/s41586-020-2405-7.pdf target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/nature-20/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://github.com/ImperialCollegeLondon/covid19model target=_blank rel=noopener>Code</a>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://github.com/ImperialCollegeLondon/covid19model target=_blank rel=noopener>Dataset</a>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://mrc-ide.github.io/covid19estimates target=_blank rel=noopener>Project</a>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://docs.google.com/presentation/d/1JJw-3ZVZvzcUsgbkzW-8BQ0T1Zvy2cHvVOTGbKPaywI/edit?usp=sharing" target=_blank rel=noopener>Slides</a>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://mrc-ide.github.io/covid19estimates/#/animations target=_blank rel=noopener>Video</a>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://www.nature.com/articles/s41586-020-2405-7 target=_blank rel=noopener>Source Document</a>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1038/s41586-020-2405-7%20 target=_blank rel=noopener>DOI</a>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-020-2405-7/MediaObjects/41586_2020_2405_MOESM1_ESM.pdf target=_blank rel=noopener>Supplementary Material</a></div></div><div class=card-simple><div class=article-metadata><div><span><a href=/authors/swapnil-mishra/>Swapnil Mishra</a></span>, <span><a href=/authors/marian-andrei-rizoiu/>Marian-Andrei Rizoiu</a></span>, <span><a href=/authors/lexing-xie/>Lexing Xie</a></span></div><span class=article-date>April 2018</span>
<span class=middot-divider></span>
<span class=pub-publication>ICWSM</span></div><a href=/publication/icwsm-18/><img src=/publication/icwsm-18/featured_huf8275e037d7bec2d4503022bf590c876_176178_918x517_fill_q90_lanczos_smart1_3.png class=article-banner alt></a><h3 class="article-title mb-1 mt-3"><a href=/publication/icwsm-18/>Modeling Popularity in Asynchronous Social Media Streams with Recurrent Neural Networks</a></h3><div class=article-style><p>Understanding and predicting the popularity of online items is an important open problem in social media analysis. Considerable progress has been made recently in data-driven predictions, and in linking popularity to external promotions. However, the existing methods typically focus on a single source of external influence, whereas for many types of online content such as YouTube videos or news articles, attention is driven by multiple heterogeneous sources simultaneously - e.g. microblogs or traditional media coverage. Here, we propose RNN-MAS, a recurrent neural network for modeling asynchronous streams. It is a sequence generator that connects multiple streams of different granularity via joint inference. We show RNN-MAS not only to outperform the current state-of-the-art Youtube popularity prediction system by 17%, but also to capture complex dynamics, such as seasonal trends of unseen influence. We define two new metrics: promotion score quantifies the gain in popularity from one unit of promotion for a Youtube video; the loudness level captures the effects of a particular user tweeting about the video. We use the loudness level to compare the effects of a video being promoted by a single highly-followed user (in the top 1% most followed users) against being promoted by a group of mid-followed users. We find that results depend on the type of content being promoted: superusers are more successful in promoting Howto and Gaming videos, whereas the cohort of regular users are more influential for Activism videos. This work provides more accurate and explainable popularity predictions, as well as computational tools for content producers and marketers to allocate resources for promotion campaigns.</p></div><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://arxiv.org/abs/1804.02101 target=_blank rel=noopener>Preprint</a>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://arxiv.org/pdf/1804.02101.pdf target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/icwsm-18/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://github.com/computationalmedia/rnn-mas target=_blank rel=noopener>Code</a>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://github.com/computationalmedia/rnn-mas target=_blank rel=noopener>Dataset</a></div></div><div class=card-simple><div class=article-metadata><div><span><a href=/authors/swapnil-mishra/>Swapnil Mishra</a></span>, <span><a href=/authors/marian-andrei-rizoiu/>Marian-Andrei Rizoiu</a></span>, <span><a href=/authors/lexing-xie/>Lexing Xie</a></span></div><span class=article-date>October 2016</span>
<span class=middot-divider></span>
<span class=pub-publication>CIKM</span></div><a href=/publication/cikm-16/><img src=/publication/cikm-16/featured_hu9870edf826a6d4733236bcc544d44b0c_267602_918x517_fill_q90_lanczos_smart1_3.png class=article-banner alt></a><h3 class="article-title mb-1 mt-3"><a href=/publication/cikm-16/>Feature Driven and Point Process Approaches for Popularity Prediction</a></h3><div class=article-style><p>Predicting popularity, or the total volume of information outbreaks, is an important subproblem for understanding collective behavior in networks. Each of the two main types of recent approaches to the problem, feature-driven and generative models, have desired qualities and clear limitations. This paper bridges the gap between these solutions with a new hybrid approach and a new performance benchmark. We model each social cascade with a marked Hawkes self-exciting point process, and estimate the content virality, memory decay, and user influence. We then learn a predictive layer for popularity prediction using a collection of cascade history. To our surprise, Hawkes process with a predictive overlay outperform recent feature-driven and generative approaches on existing tweet data [44] and a new public benchmark on news tweets. We also found that a basic set of user features and event time summary statistics performs competitively in both classification and regression tasks, and that adding point process information to the feature set further improves predictions. From these observations, we argue that future work on popularity prediction should compare across feature-driven and generative modeling approaches in both classification and regression tasks.</p></div><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://arxiv.org/abs/1608.04862 target=_blank rel=noopener>Preprint</a>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://arxiv.org/pdf/1608.04862.pdf target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/cikm-16/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://git.io/v6rIN target=_blank rel=noopener>Code</a>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://git.io/v6rIN target=_blank rel=noopener>Dataset</a>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=http://cm.cecs.anu.edu.au/post/fdhawkesforpopularity/ target=_blank rel=noopener>Project</a>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=http://cm.cecs.anu.edu.au/documents/smishra_cikm16_presentation.pdf target=_blank rel=noopener>Slides</a>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1145/2983323.2983812 target=_blank rel=noopener>DOI</a></div></div><div class=card-simple><div class=article-metadata><div><span><a href=/authors/wray-l.-buntine/>Wray L. Buntine</a></span>, <span><a href=/authors/swapnil-mishra/>Swapnil Mishra</a></span></div><span class=article-date>August 2014</span>
<span class=middot-divider></span>
<span class=pub-publication>KDD</span></div><a href=/publication/kdd-14/><img src=/publication/kdd-14/featured_hu4dfd8ffd83e04be409cb30321d35533d_124152_918x517_fill_q90_lanczos_smart1_3.png class=article-banner alt></a><h3 class="article-title mb-1 mt-3"><a href=/publication/kdd-14/>Experiments with non-parametric topic models</a></h3><div class=article-style><p>In topic modelling, various alternative priors have been developed, for instance asymmetric and symmetric priors for the document-topic and topic-word matrices respectively, the hierarchical Dirichlet process prior for the document-topic matrix and the hierarchical Pitman-Yor process prior for the topic-word matrix. For information retrieval, language models exhibiting word burstiness are important. Indeed, this burstiness effect has been show to help topic models as well, and this requires additional word probability vectors for each document. Here we show how to combine these ideas to develop high-performing non-parametric topic models exhibiting burstiness based on standard Gibbs sampling. Experiments are done to explore the behavior of the models under different conditions and to compare the algorithms with previously published. The full non-parametric topic models with burstiness are only a small factor slower than standard Gibbs sampling for LDA and require double the memory, making them very competitive. We look at the comparative behaviour of different models and present some experimental insights.</p></div><div class=btn-links><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://dl.acm.org/citation.cfm?id=2623691" target=_blank rel=noopener>Preprint</a>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=/files/p881-buntine.pdf? target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/kdd-14/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://github.com/wbuntine/topic-models target=_blank rel=noopener>Code</a>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://topicmodels.org/2014/08/28/kdd-2014/ target=_blank rel=noopener>Project</a>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://topicmodelsdotorg.files.wordpress.com/2014/10/kdd14talk.pdf target=_blank rel=noopener>Slides</a>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=http://videolectures.net/kdd2014_buntine_mishra_topic_models/ target=_blank rel=noopener>Video</a>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1145/2623330.2623691 target=_blank rel=noopener>DOI</a></div></div></div></div></div></section><section id=publications class="home-section wg-pages"><div class=container><div class=row><div class="col-12 col-lg-4 section-heading"><h1>Recent Publications</h1></div><div class="col-12 col-lg-8"><div class=pub-list-item style=margin-bottom:1rem><i class="far fa-file-alt pub-icon" aria-hidden=true></i>
<span class="article-metadata li-cite-author"><span><a href=/authors/george-altman/>George Altman</a></span>, <span><a href=/authors/janvi-ahuja/>Janvi Ahuja</a></span>, <span><a href=/authors/joshua-teperowski-monrad/>Joshua Teperowski Monrad</a></span>, <span><a href=/authors/gurpreet-dhaliwal/>Gurpreet Dhaliwal</a></span>, <span><a href=/authors/charlie-rogers-smith/>Charlie Rogers-Smith</a></span>, <span><a href=/authors/gavin-leech/>Gavin Leech</a></span>, <span><a href=/authors/benedict-snodin/>Benedict Snodin</a></span>, <span><a href=/authors/jonas-b-sandbrink/>Jonas B Sandbrink</a></span>, <span><a href=/authors/lukas-finnveden/>Lukas Finnveden</a></span>, <span><a href=/authors/alexander-john-norman/>Alexander John Norman</a></span>, <span><a href=/authors/others/>others</a></span></span>
(2022).
<a href=/publication/altman-2022-dataset/>A dataset of non-pharmaceutical interventions on SARS-CoV-2 in Europe</a>.
<em>Scientific Data</em>.<p><button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/altman-2022-dataset/cite.bib>
Cite</button></p></div><div class=pub-list-item style=margin-bottom:1rem><i class="far fa-file-alt pub-icon" aria-hidden=true></i>
<span class="article-metadata li-cite-author"><span><a href=/authors/seth-flaxman/>Seth Flaxman</a></span>, <span><a href=/authors/charles-whittaker/>Charles Whittaker</a></span>, <span><a href=/authors/elizaveta-semenova/>Elizaveta Semenova</a></span>, <span><a href=/authors/theo-rashid/>Theo Rashid</a></span>, <span><a href=/authors/robbie-parks/>Robbie Parks</a></span>, <span><a href=/authors/alexandra-blenkinsop/>Alexandra Blenkinsop</a></span>, <span><a href=/authors/h-juliette-t-unwin/>H Juliette T Unwin</a></span>, <span><a href=/authors/swapnil-mishra/>Swapnil Mishra</a></span>, <span><a href=/authors/samir-bhatt/>Samir Bhatt</a></span>, <span><a href=/authors/deepti-gurdasani/>Deepti Gurdasani</a></span>, <span><a href=/authors/others/>others</a></span></span>
(2022).
<a href=/publication/flaxman-2022-covid/>Covid-19 is a leading cause of death in children and young people ages 0-19 years in the United States</a>.
<em>medRxiv</em>.<p><button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/flaxman-2022-covid/cite.bib>
Cite</button></p></div><div class=pub-list-item style=margin-bottom:1rem><i class="far fa-file-alt pub-icon" aria-hidden=true></i>
<span class="article-metadata li-cite-author"><span><a href=/authors/xenia-miscouridou/>Xenia Miscouridou</a></span>, <span><a href=/authors/samir-bhatt/>Samir Bhatt</a></span>, <span><a href=/authors/george-mohler/>George Mohler</a></span>, <span><a href=/authors/seth-flaxman/>Seth Flaxman</a></span>, <span><a href=/authors/swapnil-mishra/>Swapnil Mishra</a></span></span>
(2022).
<a href=/publication/miscouridou-2022-cox/>Cox-Hawkes: doubly stochastic spatiotemporal Poisson processes</a>.
<em>arXiv preprint arXiv:2210.11844</em>.<p><button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/miscouridou-2022-cox/cite.bib>
Cite</button></p></div><div class=pub-list-item style=margin-bottom:1rem><i class="far fa-file-alt pub-icon" aria-hidden=true></i>
<span class="article-metadata li-cite-author"><span><a href=/authors/nicholas-f-brazeau/>Nicholas F Brazeau</a></span>, <span><a href=/authors/robert-verity/>Robert Verity</a></span>, <span><a href=/authors/sara-jenks/>Sara Jenks</a></span>, <span><a href=/authors/han-fu/>Han Fu</a></span>, <span><a href=/authors/charles-whittaker/>Charles Whittaker</a></span>, <span><a href=/authors/peter-winskill/>Peter Winskill</a></span>, <span><a href=/authors/ilaria-dorigatti/>Ilaria Dorigatti</a></span>, <span><a href=/authors/patrick-gt-walker/>Patrick GT Walker</a></span>, <span><a href=/authors/steven-riley/>Steven Riley</a></span>, <span><a href=/authors/ricardo-p-schnekenberg/>Ricardo P Schnekenberg</a></span>, <span><a href=/authors/others/>others</a></span></span>
(2022).
<a href=/publication/brazeau-2022-estimating/>Estimating the COVID-19 infection fatality ratio accounting for seroreversion using statistical modelling</a>.
<em>Communications medicine</em>.<p><button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/brazeau-2022-estimating/cite.bib>
Cite</button></p></div><div class=pub-list-item style=margin-bottom:1rem><i class="far fa-file-alt pub-icon" aria-hidden=true></i>
<span class="article-metadata li-cite-author"><span><a href=/authors/elizaveta-semenova/>Elizaveta Semenova</a></span>, <span><a href=/authors/yidan-xu/>Yidan Xu</a></span>, <span><a href=/authors/adam-howes/>Adam Howes</a></span>, <span><a href=/authors/theo-rashid/>Theo Rashid</a></span>, <span><a href=/authors/samir-bhatt/>Samir Bhatt</a></span>, <span><a href=/authors/swapnil-mishra/>Swapnil Mishra</a></span>, <span><a href=/authors/seth-flaxman/>Seth Flaxman</a></span></span>
(2022).
<a href=/publication/semenova-2022-priorvae/>PriorVAE: encoding spatial priors with variational autoencoders for small-area estimation</a>.
<em>Journal of the Royal Society Interface</em>.<p><button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/semenova-2022-priorvae/cite.bib>
Cite</button></p></div><div class=pub-list-item style=margin-bottom:1rem><i class="far fa-file-alt pub-icon" aria-hidden=true></i>
<span class="article-metadata li-cite-author"><span><a href=/authors/andrea-brizzi/>Andrea Brizzi</a></span>, <span><a href=/authors/charles-whittaker/>Charles Whittaker</a></span>, <span><a href=/authors/luciana-servo/>Luciana Servo</a></span>, <span><a href=/authors/iwona-hawryluk/>Iwona Hawryluk</a></span>, <span><a href=/authors/carlos-a-prete/>Carlos A Prete</a></span>, <span><a href=/authors/william-m-de-souza/>William M de Souza</a></span>, <span><a href=/authors/renato-s-aguiar/>Renato S Aguiar</a></span>, <span><a href=/authors/leonardo-jt-araujo/>Leonardo JT Araujo</a></span>, <span><a href=/authors/leonardo-s-bastos/>Leonardo S Bastos</a></span>, <span><a href=/authors/alexandra-blenkinsop/>Alexandra Blenkinsop</a></span>, <span><a href=/authors/others/>others</a></span></span>
(2022).
<a href=/publication/brizzi-2022-spatial/>Spatial and temporal fluctuations in COVID-19 fatality rates in Brazilian hospitals</a>.
<em>Nature medicine</em>.<p><button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/brizzi-2022-spatial/cite.bib>
Cite</button></p></div><div class=pub-list-item style=margin-bottom:1rem><i class="far fa-file-alt pub-icon" aria-hidden=true></i>
<span class="article-metadata li-cite-author"><span><a href=/authors/matthew-j-penn/>Matthew J Penn</a></span>, <span><a href=/authors/daniel-j-laydon/>Daniel J Laydon</a></span>, <span><a href=/authors/joseph-penn/>Joseph Penn</a></span>, <span><a href=/authors/charles-whittaker/>Charles Whittaker</a></span>, <span><a href=/authors/christian-morgenstern/>Christian Morgenstern</a></span>, <span><a href=/authors/oliver-ratmann/>Oliver Ratmann</a></span>, <span><a href=/authors/swapnil-mishra/>Swapnil Mishra</a></span>, <span><a href=/authors/mikko-s-pakkanen/>Mikko S Pakkanen</a></span>, <span><a href=/authors/christl-a-donnelly/>Christl A Donnelly</a></span>, <span><a href=/authors/samir-bhatt/>Samir Bhatt</a></span></span>
(2022).
<a href=/publication/penn-2022-uncertainty/>The uncertainty of infectious disease outbreaks is underestimated</a>.
<em>arXiv preprint arXiv:2210.14221</em>.<p><button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/penn-2022-uncertainty/cite.bib>
Cite</button></p></div><div class=pub-list-item style=margin-bottom:1rem><i class="far fa-file-alt pub-icon" aria-hidden=true></i>
<span class="article-metadata li-cite-author"><span><a href=/authors/swapnil-mishra/>Swapnil Mishra</a></span>, <span><a href=/authors/seth-flaxman/>Seth Flaxman</a></span>, <span><a href=/authors/tresnia-berah/>Tresnia Berah</a></span>, <span><a href=/authors/harrison-zhu/>Harrison Zhu</a></span>, <span><a href=/authors/mikko-pakkanen/>Mikko Pakkanen</a></span>, <span><a href=/authors/samir-bhatt/>Samir Bhatt</a></span></span>
(2022).
<a href=/publication/mishra-2022-pi/>π VAE: a stochastic process prior for Bayesian deep learning with MCMC</a>.
<em>Statistics and Computing</em>.<p><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://link.springer.com/content/pdf/10.1007/s11222-022-10151-w.pdf target=_blank rel=noopener>PDF</a>
<button type=button class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal" data-filename=/publication/mishra-2022-pi/cite.bib>
Cite</button>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://github.com/MLGlobalHealth/pi-vae%20 target=_blank rel=noopener>Code</a>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://github.com/MLGlobalHealth/pi-vae%20 target=_blank rel=noopener>Dataset</a>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.1007/s11222-022-10151-w target=_blank rel=noopener>DOI</a>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://link.springer.com/article/10.1007/s11222-022-10151-w#appendices target=_blank rel=noopener>Supplementary Material</a></p></div><div class=pub-list-item style=margin-bottom:1rem><i class="far fa-file-alt pub-icon" aria-hidden=true></i>
<span class="article-metadata li-cite-author"><span><a href=/authors/petra-mlcochova/>Petra Mlcochova</a></span>, <span><a href=/authors/steven-a-kemp/>Steven A Kemp</a></span>, <span><a href=/authors/mahesh-shanker-dhar/>Mahesh Shanker Dhar</a></span>, <span><a href=/authors/others/>others</a></span>, <span><a href=/authors/swapnil-mishra/>Swapnil Mishra</a></span>, <span><a href=/authors/charles-whittaker/>Charles Whittaker</a></span>, <span><a href=/authors/thomas-mellan/>Thomas Mellan</a></span>, <span><a href=/authors/others/>others</a></span>, <span><a href=/authors/samir-bhatt/>Samir Bhatt</a></span>, <span><a href=/authors/seth-flaxman/>Seth Flaxman</a></span>, <span><a href=/authors/leo-c-james/>Leo C James</a></span>, <span><a href=/authors/davide-corti/>Davide Corti</a></span>, <span><a href=/authors/luca-piccoli/>Luca Piccoli</a></span>, <span><a href=/authors/wendy-s-barclay/>Wendy S Barclay</a></span>, <span><a href=/authors/partha-rakshit/>Partha Rakshit</a></span>, <span><a href=/authors/anurag-agrawal/>Anurag Agrawal</a></span>, <span><a href=/authors/ravindra-k-gupta/>Ravindra K Gupta</a></span></span>
(2021).
<a href=/publication/nature-delta-21/>SARS-CoV-2 B. 1.617. 2 Delta variant replication and immune evasion</a>.
Nature (2021).<p><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://www.nature.com/articles/s41586-021-03944-y.pdf target=_blank rel=noopener>PDF</a>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://github.com/Steven-Kemp/hospital_india/tree/main/consensus_fasta target=_blank rel=noopener>Code</a>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://github.com/Steven-Kemp/hospital_india/tree/main/consensus_fasta target=_blank rel=noopener>Dataset</a>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://www.nature.com/articles/s41586-021-03944-y target=_blank rel=noopener>Source Document</a>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/https://doi.org/10.1038/s41586-021-03944-y target=_blank rel=noopener>DOI</a></p></div><div class=pub-list-item style=margin-bottom:1rem><i class="far fa-file-alt pub-icon" aria-hidden=true></i>
<span class="article-metadata li-cite-author"><span><a href=/authors/mahesh-s.-dhar/>Mahesh S. Dhar</a></span>, <span><a href=/authors/robin-marwal/>Robin Marwal</a></span>, <span><a href=/authors/radhakrishnan-vs/>Radhakrishnan VS</a></span>, <span><a href=/authors/kalaiarasan-ponnusamy/>Kalaiarasan Ponnusamy</a></span>, <span><a href=/authors/bani-jolly/>Bani Jolly</a></span>, <span><a href=/authors/rahul-c.-bhoyar/>Rahul C. Bhoyar</a></span>, <span><a href=/authors/viren-sardana/>Viren Sardana</a></span>, <span><a href=/authors/salwa-naushin/>Salwa Naushin</a></span>, <span><a href=/authors/mercy-rophina/>Mercy Rophina</a></span>, <span><a href=/authors/thomas-a.-mellan/>Thomas A. Mellan</a></span>, <span><a href=/authors/swapnil-mishra/>Swapnil Mishra</a></span>, <span><a href=/authors/charles-whittaker/>Charles Whittaker</a></span>, <span><a href=/authors/others/>others</a></span>, <span><a href=/authors/seth-flaxman/>Seth Flaxman</a></span>, <span><a href=/authors/samir-bhatt/>Samir Bhatt</a></span>, <span><a href=/authors/rajesh-pandey/>Rajesh Pandey</a></span>, <span><a href=/authors/debasis-dash/>Debasis Dash</a></span>, <span><a href=/authors/mohammed-faruq/>Mohammed Faruq</a></span>, <span><a href=/authors/hemlata-lall/>Hemlata Lall</a></span>, <span><a href=/authors/hema-gogia/>Hema Gogia</a></span>, <span><a href=/authors/preeti-madan/>Preeti Madan</a></span>, <span><a href=/authors/sanket-kulkarni/>Sanket Kulkarni</a></span>, <span><a href=/authors/himanshu-chauhan/>Himanshu Chauhan</a></span>, <span><a href=/authors/shantanu-sengupta/>Shantanu Sengupta</a></span>, <span><a href=/authors/sandhya-kabra/>Sandhya Kabra</a></span>, <span><a href=/authors/insacog/>INSACOG</a></span>, <span><a href=/authors/ravindra-k.-gupta/>Ravindra K. Gupta</a></span>, <span><a href=/authors/sujeet-k.-singh/>Sujeet K. Singh</a></span>, <span><a href=/authors/anurag-agrawal/>Anurag Agrawal</a></span>, <span><a href=/authors/partha-rakshit/>Partha Rakshit</a></span></span>
(2021).
<a href=/publication/science-delta-21/>Genomic characterization and epidemiology of an emerging SARS-CoV-2 variant in Delhi, India</a>.
Science (2021).<p><a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://www.science.org/doi/reader/10.1126/science.abj9932 target=_blank rel=noopener>PDF</a>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.5281/zenodo.5524442 target=_blank rel=noopener>Code</a>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/10.5281/zenodo.5521073 target=_blank rel=noopener>Dataset</a>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://www.science.org/doi/10.1126/science.abj9932 target=_blank rel=noopener>Source Document</a>
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href=https://doi.org/https://doi.org/10.1126/science.abj9932 target=_blank rel=noopener>DOI</a></p></div><div class=see-all><a href=/publication/>See all publications
<i class="fas fa-angle-right"></i></a></div></div></div></div></section><section id=grants class="home-section wg-blank"><div class=container><div class=row><div class="col-12 col-lg-4 section-heading"><h1>Grants</h1></div><div class="col-12 col-lg-8"><h3 id=__digital-technology-development-award-wellcome-trust__><strong>Digital Technology Development Award Wellcome Trust</strong></h3><ul><li><strong>Year:</strong> 2022-2027</li><li><strong>Granting Agency:</strong> Imperial College London</li><li><strong>Amount:</strong> $612,665 USD</li><li><strong>Role:</strong> Co-Principal Investigator</li></ul><h3 id=__imperial-covid-19-response-fund__><strong>Imperial COVID-19 Response Fund</strong></h3><ul><li><strong>Year:</strong> 2020-2021</li><li><strong>Granting Agency:</strong> Imperial College London</li><li><strong>Amount:</strong> £46,975 GBP</li><li><strong>Role:</strong> Co-Investigator</li></ul><h3 id=__microsoft-ai-for-health-program__><strong>Microsoft AI for Health program</strong></h3><ul><li><strong>Year:</strong> 2020-2021</li><li><strong>Granting Agency:</strong> Microsoft</li><li><strong>Amount:</strong> $130,000 USD</li><li><strong>Role:</strong> Co-Investigator</li></ul><h3 id=__amazon-aws-compute-grant__><strong>Amazon AWS Compute Grant</strong></h3><ul><li><strong>Year:</strong> 2020-2021</li><li><strong>Granting Agency:</strong> Amazon</li><li><strong>Amount:</strong> $40,000 USD</li><li><strong>Role:</strong> Co-Investigator</li></ul></div></div></div></section><section id=teaching class="home-section wg-pages"><div class=container><div class=row><div class="col-12 col-lg-4 section-heading"><h1>Teaching</h1></div><div class="col-12 col-lg-8"><p>I was a teaching instructor for the following courses at The Australian National University:</p><ul><li>COMP4650/6490: Document Analysis (2013-2018)</li><li>COMP2420/6420 (Course designer and Lecturer): Introduction to Data Management, Analysis and Security (2018)</li><li>COMP2410/6340: Networked Information Systems (2014, 2017, 2018)</li><li>COMP7230(Co-lecturer): Introduction to Programming for Data Scientists (2017)</li><li>COMP1730/6730: Programming for Scientist (2014, 2015)</li></ul></div></div></div></section><section id=contact class="home-section wg-contact"><div class=container><div class="row contact-widget"><div class="col-12 col-lg-4 section-heading"><h1>Contact</h1></div><div class="col-12 col-lg-8"><ul class=fa-ul><li><i class="fa-li fas fa-envelope fa-2x" aria-hidden=true></i>
<span id=person-email><a href=mailto:swapnilmishra%20dot%20anu%20at%20gmail%20dot%20com>swapnilmishra dot anu at gmail dot com</a></span></li><li><i class="fa-li fas fa-phone fa-2x" aria-hidden=true></i>
<span id=person-telephone><a href=tel:+45%2035%2032%2022%2078>+45 35 32 22 78</a></span></li><li><i class="fa-li fas fa-map-marker fa-2x" aria-hidden=true></i>
<span id=person-address>Bartholinsgade 6Q, 2. sal, København K, 1356, DENMARK</span></li><li><i class="fa-li fas fa-compass fa-2x" aria-hidden=true></i>
<span>Enter the second door in the building and ring the bell on the door to the left.</span></li><li><i class="fa-li fas fa-clock fa-2x" aria-hidden=true></i>
<span>Monday-Friday 09:00 to 15:00</span></li><li><i class="fa-li fab fa-twitter fa-2x" aria-hidden=true></i>
<a href=https://twitter.com/creswapi target=_blank rel=noopener>DM Me</a></li><li><i class="fa-li fab fa-skype fa-2x" aria-hidden=true></i>
<a href=skype:misraswapnil?call>Skype Me</a></li></ul><div class=d-none><input id=map-provider value=1>
<input id=map-lat value=55.687496>
<input id=map-lng value=12.567644>
<input id=map-dir value="Bartholinsgade 6Q, 2. sal, København K, 1356, DENMARK">
<input id=map-zoom value=15>
<input id=map-api-key value=AIzaSyDGqCHkZIA7ZavBfuM-EczgZCVaJvJqUHs></div><div id=map></div></div></div></div></section><script src=https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin=anonymous></script>
<script src=https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.4/imagesloaded.pkgd.min.js integrity="sha256-lqvxZrPLtfffUl2G/e7szqSvPBILGbwmsGE1MKlOi0Q=" crossorigin=anonymous></script>
<script src=https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.6/isotope.pkgd.min.js integrity="sha256-CBrpuqrMhXwcLLUd5tvQ4euBHCdh7wGlDfNz8vbu/iI=" crossorigin=anonymous></script>
<script src=https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.js integrity="sha256-yt2kYMy0w8AbtF89WXb2P1rfjcP/HTHLT7097U8Y5b8=" crossorigin=anonymous></script>
<script src=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.10/highlight.min.js integrity="sha256-1zu+3BnLYV9LdiY85uXMzii3bdrkelyp37e0ZyTAQh0=" crossorigin=anonymous></script>
<script src=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.10/languages/r.min.js></script>
<script async defer src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDGqCHkZIA7ZavBfuM-EczgZCVaJvJqUHs"></script>
<script src=https://cdnjs.cloudflare.com/ajax/libs/gmaps.js/0.4.25/gmaps.min.js integrity="sha256-7vjlAeb8OaTrCXZkCNun9djzuB2owUsaO72kXaFDBJs=" crossorigin=anonymous></script>
<script>hljs.initHighlightingOnLoad()</script><script>const search_config={indexURI:"/index.json",minLength:1,threshold:.3},i18n={no_results:"No results found",placeholder:"Search...",results:"results found"},content_type={post:"Posts",project:"Projects",publication:"Publications",talk:"Talks"}</script><script id=search-hit-fuse-template type=text/x-template>
      <div class="search-hit" id="summary-{{key}}">
      <div class="search-hit-content">
        <div class="search-hit-name">
          <a href="{{relpermalink}}">{{title}}</a>
          <div class="article-metadata search-hit-type">{{type}}</div>
          <p class="search-hit-description">{{snippet}}</p>
        </div>
      </div>
      </div>
    </script><script src=https://cdnjs.cloudflare.com/ajax/libs/fuse.js/3.2.1/fuse.min.js integrity="sha256-VzgmKYmhsGNNN4Ph1kMW+BjoYJM2jV5i4IlFoeZA9XI=" crossorigin=anonymous></script>
<script src=https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/jquery.mark.min.js integrity="sha256-4HLtjeVgH0eIB3aZ9mLYF6E8oU5chNdjU6p6rrXpl9U=" crossorigin=anonymous></script>
<script src=/js/academic.min.01f68d44d80310e669a1731b68a35481.js></script><div class=container><footer class=site-footer><p class=powered-by>©2023 &#183;
Powered by the
<a href=https://sourcethemes.com/academic/ target=_blank rel=noopener>Academic theme</a> for
<a href=https://gohugo.io target=_blank rel=noopener>Hugo</a>.
<span class=float-right aria-hidden=true><a href=# class=back-to-top><span class=button_icon><i class="fas fa-chevron-up fa-2x"></i></span></a></span></p></footer></div><div id=modal class="modal fade" role=dialog><div class=modal-dialog><div class=modal-content><div class=modal-header><h5 class=modal-title>Cite</h5><button type=button class=close data-dismiss=modal aria-label=Close>
<span aria-hidden=true>&#215;</span></button></div><div class=modal-body><pre><code class="tex hljs"></code></pre></div><div class=modal-footer><a class="btn btn-outline-primary my-1 js-copy-cite" href=# target=_blank><i class="fas fa-copy"></i> Copy</a>
<a class="btn btn-outline-primary my-1 js-download-cite" href=# target=_blank><i class="fas fa-download"></i> Download</a><div id=modal-error></div></div></div></div></div></body></html>